app/src/main/java/appricottsoftware/clarity/fragments/BrowseFragment.java

```diff
@@ -0,0 +1,31 @@
+package appricottsoftware.clarity.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import appricottsoftware.clarity.R;
+import butterknife.ButterKnife;
+
+/**
+ * Created by karen on 1/19/18.
+ */
+
+public class BrowseFragment extends Fragment {
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_browse, container, false);
+        ButterKnife.bind(this, view);
+        return view;
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+
+        // Initialize view lookups, listeners
+    }
```

Comment:
Is this function done? If not, add a "TODO"

Reply:
No it's not done. This is an app skeleton. All the files are "TODO".